### PR TITLE
Changed the release version from tier-0 to rc

### PR DESCRIPTION
**Pull request by krishnaramaswamy**
# Description

Fixing the rbd upgrade from 5.x to 6.x Test Suite Failure.
<details>

<summary>click to expand checklist</summary>

</details>

### CODE REVIEW

File: suites/quincy/rbd/tier-2_rbd_upgrade_5x_to_6x.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ tests:
                   service: cephadm
                   args:
                     rhcs-version: 5.3
-                    release: "tier-0"
+                    release: "rc"
```
Comment (sunilangadi2) [inline suites/quincy/rbd/tier-2_rbd_upgrade_5x_to_6x.yaml:29]: Thanks @krishnaramaswamy for the PR
can you please test these changes and update test result?

**Comment by sunilangadi2** [comment]
@manasagowri i think this issue https://issues.redhat.com/browse/RHCEPHQE-11570
resolves this fix.Please review it.

**Comment by manasagowri** [comment]
> @manasagowri i think this issue https://issues.redhat.com/browse/RHCEPHQE-11570 resolves this fix.Please review it.

@sunilangadi2 Yes this resolves the 5x_to_6x upgrade issue. Once we get success logs we can merge it.

**Comment by krishnaramaswamy** [comment]
Hi @sunilangadi2 ,

 i have updated the secondary cluster ceph-rbd2 in line 73 as release: "rc"

**Comment by tintumathew10** [comment]
[cephci-run-VN5K3L.zip](https://github.com/red-hat-storage/cephci/files/12937528/cephci-run-VN5K3L.zip)

**Review by sunilangadi2 — APPROVED** [review]
thanks @krishnaramaswamy LGTM.

**Pull request merged**